app/models/itsaStatus/ITSAStatusResponse.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package models.itsaStatus
+
+import play.api.libs.json.{Format, Json}
+
+sealed trait ITSAStatusResponse
+
+case class ITSAStatusResponseModel(taxYear: String,
+                                   itsaStatusDetails: Option[List[StatusDetail]] = None) extends ITSAStatusResponse
```

Comment:
can we have more than one status for a year?

Reply:
Its according to schema